docs/src/Experimental/galois.md

```diff
@@ -0,0 +1,207 @@
+```@meta
+CurrentModule = Oscar
+```
+
+```@setup oscar
+using Oscar
+```
+
+```@contents
+Pages = ["galois.md"]
+```
+
+# Galois Theory
+
+Let `K` be a finite (separable) extension of `k`. Then, in contrast to most of 
+the literature we distinguish two concepts
+
+ - the *automorphism group*
+ - the *Galois group*
+ 
+The automorphism group deals with the actual automorphism of `K` fixing `k`
+and thus is, in general trivial. Access to via two constructions:
+
+ - a list of all automorphisms (usually only the identity)
+ - the group of automorphisms, returned as an abstact group and
+   a map linking group elements to actual automorphisms
+
+On the other hand, the Galois group is isomorphic to the automorphism
+group of the normal closure and is explicitly given as a group of
+permutations of the roots  of the defining polynomial. Thus even
+in the case of `K` over `k` being normal, elements of the
+Galois group do not immediately give automorphisms at all.
+
+Currently, the computation of Galois groups is possible for
+ 
+  - `K` a simple extension of the rationals (`AnticNumberField`)
+  - `K` a simple extension of an `AnticNumberField`
```

Comment:
Instead of mentioning the type perhaps it would be more helpful to mention the function(s) that produce fields of these types?

Reply:
the function is in all cases `number_field` - or possibly, also, in the future, `extension_field`, `ResidueField`, `quo(NumberField, ...)` so this is tricky